cmd/cstorvolumeclaim/cstorvolumeclaim.go

```diff
@@ -367,6 +370,8 @@ func createCVR(
 			WithFinalizers(getCVRFinalizer()).
 			WithTargetIP(service.Spec.ClusterIP).
 			WithCapacity(volume.Spec.Capacity.String()).
+			WithNewVersion(version.GetVersion()).
```

Comment:
is setting newVersion handled for replicas created from CAS templates?

Reply:
Yes